app/forms/disk-create.tsx

```diff
@@ -79,6 +80,19 @@ export function CreateDiskSideModalForm({
   })
 
   const form = useForm({ defaultValues })
+  const control = form.control
+  const { project } = useProjectSelector()
+  const projectImages = useApiQuery('imageList', { query: { project } })
+  const siloImages = useApiQuery('imageList', {})
+
+  // put project images first because if there are any, there probably aren't
+  // very many and they're probably relevant
+  const images = [...(projectImages.data?.items || []), ...(siloImages.data?.items || [])]
```

Comment:
I know this is how it was before, but let's put this in a `useMemo(..., [projectImages.data, siloImages.data])` — constructing fresh arrays on every render will trigger extra renders further down the chain every time.